FWCore/Framework/test/eventsetup_t.cppunit.cc

```diff
@@ -590,77 +590,145 @@ namespace {
     ESGetToken<edm::eventsetup::test::DummyData, edm::DefaultRecord> m_token;
   };
 
-  class ConsumesProducer : public ESProducer {
-  public:
-    ConsumesProducer() : token_{setWhatProduced(this, "consumes").consumes<edm::eventsetup::test::DummyData>()} {}
-    std::unique_ptr<edm::eventsetup::test::DummyData> produce(const DummyRecord& iRecord) {
-      auto const& data = iRecord.get(token_);
-      return std::make_unique<edm::eventsetup::test::DummyData>(data);
-    }
+  //This just tests that the constructs will properly compile
+  class [[maybe_unused]] EDConsumesCollectorConsumer
+      : public edm::EDConsumerBase{EDConsumesCollectorConsumer(){using edm::eventsetup::test::DummyData;
+  {
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token1(
+        consumesCollector().esConsumes<DummyData, edm::DefaultRecord>());
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token2(
+        consumesCollector().esConsumes<DummyData, edm::DefaultRecord>(edm::ESInputTag("Blah")));
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token3(
+        consumesCollector().esConsumes<DummyData, edm::DefaultRecord, edm::Transition::BeginRun>());
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token4(
+        consumesCollector().esConsumes<DummyData, edm::DefaultRecord, edm::Transition::BeginRun>(
+            edm::ESInputTag("Blah")));
+  }
+  {
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token1(consumesCollector().esConsumes());
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token2(
+        consumesCollector().esConsumes(edm::ESInputTag("Blah")));
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token3(
+        consumesCollector().esConsumes<edm::Transition::BeginRun>());
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token4(
+        consumesCollector().esConsumes<edm::Transition::BeginRun>(edm::ESInputTag("Blah")));
+  }
+  {
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token1;
+    token1 = consumesCollector().esConsumes();
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token2;
+    token2 = consumesCollector().esConsumes(edm::ESInputTag("Blah"));
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token3;
+    token3 = consumesCollector().esConsumes<edm::Transition::BeginRun>();
+    [[maybe_unused]] edm::ESGetToken<DummyData, edm::DefaultRecord> token4;
+    token4 = consumesCollector().esConsumes<edm::Transition::BeginRun>(edm::ESInputTag("Blah"));
+  }
 
-  private:
-    edm::ESGetToken<edm::eventsetup::test::DummyData, DummyRecord> token_;
-  };
+}  // namespace
+}
+;
```

Comment:
These three lines look confusing. If I read it right, the first closing curly bracket `}  // namespace` actually closes the constructor. Has this file gone through `clang-format`?

Reply:
yes, this is from clang-frormat